pkg/runtimeproxy/resexecutor/cri/container_test.go

```diff
@@ -219,3 +219,51 @@ func TestContainerResourceExecutor_UpdateRequestForUpdateContainerResourcesReque
 		assert.Equal(t, tt.wantAnnotations, tt.args.req.(*runtimeapi.UpdateContainerResourcesRequest).GetAnnotations())
 	}
 }
+
+func TestContainerResourceExecutor_ResourceCheckPoint(t *testing.T) {
```

Comment:
There are still some codes in `ResourceCheckPoint()` are not coverd by this UT, could you help add more coverage? Thanks!
For more detailed coverage report, please check: https://app.codecov.io/gh/koordinator-sh/koordinator/compare/251/changes